lib/v2/hdu/radar.js

```diff
@@ -15,5 +15,21 @@ module.exports = {
                 target: '/hdu/cs/pg',
             },
         ],
+        jwc: [
+            {
+                title: '教务处',
+                docs: 'https://docs.rsshub.app/routes/university#hang-zhou-dian-zi-ke-ji-da-xue',
+                source: '/',
+                target: '/hdu/jwc/:category',
+            },
+        ],
+        grs: [
+            {
+                title: '研究生院',
+                docs: 'https://docs.rsshub.app/routes/university#hang-zhou-dian-zi-ke-ji-da-xue',
+                source: '/',
+                target: '/hdu/grs/:category',
+            },
+        ],
```

Comment:
Please sort the entries in alphabetical order.
```suggestion
        grs: [
            {
                title: '研究生院',
                docs: 'https://docs.rsshub.app/routes/university#hang-zhou-dian-zi-ke-ji-da-xue',
                source: '/',
                target: '/hdu/grs/:category',
            },
        ],
        jwc: [
            {
                title: '教务处',
                docs: 'https://docs.rsshub.app/routes/university#hang-zhou-dian-zi-ke-ji-da-xue',
                source: '/',
                target: '/hdu/jwc/:category',
            },
        ],
```